### PR TITLE
Change stats data file path for better cachability.

### PR DESCRIPTION
**Pull request by pmclanahan**
- Put stats json files in thier own dir so we can increase cache times.
- Modify paths to files in JS and use static path from template.
- Add Unipath for cleaner and easier python path manipulation.

@jgmize r?

### CODE REVIEW

File: smithers/milhouse.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import argparse
 import json
 import logging
+import os
```
Comment (jgmize) [inline smithers/milhouse.py]: flake8 doesn't like this import.

File: requirements/mrburns.txt [hunk 2]
```diff
@@ -37,3 +37,6 @@ nose==1.3.1
 
 # sha256: mq4WtWKGak3apeiXhymrrbvtVEco2I4LnJr3sx3Qcu0
 django-nose==1.2
+
+# sha256: 7wE5rDCRO0Po04ij53blNKIvgZdZZpaNKDaewDOKabc
+Unipath==1.0
```
Comment (jgmize) [inline requirements/mrburns.txt]: I don't have an objection to Unipath, but as mentioned in https://github.com/mikeorr/Unipath#introduction you may want to consider using [pathlib](https://pypi.python.org/pypi/pathlib/) instead, which is available for python 2.7 and part of the standard library in 3.4.

**Comment by pmclanahan** [comment]
@jgmize switched to pathlib. I think we're good to go now as long as Travis agrees.

**Comment by pmclanahan** [comment]
I'll squarsh if you r+ this version.

**Pull request merged**